src/routes/users/me/payments/_post/index.ts

```diff
@@ -104,6 +137,55 @@ export default class Route extends UserRoute<{
     return { requestId };
   }
 
+  private calculateFiscalData() {
+    switch (this.fiscalProfile.fiscal_category_name) {
+      case "withholding":
+        return this.calculateFiscalDataWithholding();
+      case "witholding-extra":
+        return this.calculateFiscalDataWithholdingExtra();
+      case "vat":
+        return {
+          tax_percent: 0,
+          net: this.booty,
+          witholding: 0,
+          net_multiplier: 1.04,
+        };
+      case "company":
+        return {
+          tax_percent: 0,
+          net: this.booty,
+          witholding: 0,
+          net_multiplier: 1.02,
+        };
+      default:
+        return {
+          tax_percent: 0,
+          net: this.booty,
+          witholding: 0,
+        };
+    }
+  }
+
+  private calculateFiscalDataWithholding() {
+    return {
+      tax_percent: 20,
+      net: Math.round((this.booty + Number.EPSILON) * 80) / 100,
+      witholding: Math.round((this.booty + Number.EPSILON) * 20) / 100,
+    };
+  }
+
+  private calculateFiscalDataWithholdingExtra() {
+    const net = parseFloat(
+      (0.72 * ((this.booty + Number.EPSILON) / 1.16)).toFixed(2)
+    );
+    const witholding = this.booty - net;
+    return {
+      tax_percent: 100 - parseFloat(((100 * 0.72) / 1.16).toFixed(2)),
```

Comment:
100 - parseFloat(((100 * 0.72) / 1.16).toFixed(2)), this expression does not depend on variable values? the result is 37.93. Do you want to leave the calculation in the code ? or do you think it is better to write 37.93 directly ?